packages/analyticsdx-template-lint/src/schemas/readiness-schema.json

```diff
@@ -150,25 +150,29 @@
             "OrgDatasetRowCount",
             "AppDatasetRowCount",
             "DataCloudRowCount",
+            "ApexCallout",
             "OrgPreferenceCheck",
-            "ApexCallout"
+            "AccessCheck"
```

Comment:
qq would line 50 and line 63 also need to be updated with the addition of `accessCheck`?

Reply:
Yes, good catch. I'll fix that up.